### ISSUE TITLE
Request: Some indication of points contributed by each exercise in a module

### ISSUE THREAD

**Issue by ayaankazerouni** [OpenDSA/OpenDSA#367]
When using OpenDSA over LTI, a given module might contain several exercises on the page. Because of the way Canvas does things, all of those exercises are combined into a single "assignment" in Canvas, and a single grade for the student. Unfortunately, this makes it hard to tell how many points are contributed by each exercise in a given module (or equivalently in a given Canvas assignment).

For example, the module on Comparisons of List Implementations has exercises for "Space Comparison" (worth 1 point) and "Time Comparison" (worth 2 points) for a total of 3 points. I know these point values because I went digging into book configurations, but it would be good to surface those point values to the presentation in Canvas.

**Comment by cashaffer** [comment]
You are asking here for a presentation to the reader (student) in the module page?

**Comment by ayaankazerouni** [comment]
Yes, to the student (and to the instructor as well I suppose).

**Comment by cashaffer** [comment]
While we are considering this, here is a related issue: Occasionally, a student appears not to notice that KA question banks require multiple points to complete the exercise. Some additional information could be provided. For example, under the line that says "1 of 5", there could be another line (in red until the exercise is complete) that says "You have to earn 4 more points to complete this exercise."

**Comment by ayaankazerouni** [comment]
Yes, that would be good idea as well. It's not always clear, for example, that using a hint to arrive at an answer means that you DON'T get credit for that answer. Having something a bit "louder" there would be great.
